### PR TITLE
Read simc dumps

### PR DESCRIPTION
**Pull request by prsauer**

### CODE REVIEW

File: packages/shared/src/components/CombatReport/CombatReplay/ReplayUnitFrames/UnitFrame.tsx [hunk 1]
```diff
@@ -213,10 +213,18 @@ export const UnitFrame = (props: IProps) => {
     }
     if (sd['2050']) {
       // 235587 Miracle Worker (Holy Word: Serenity)
-      if (props.unit.info?.pvpTalents.includes('235587')) {
+      if (props.unit.info?.talents.find((t) => t?.id2 === 103737)) {
```
Comment (atinylittleshell) [inline packages/shared/src/components/CombatReport/CombatReplay/ReplayUnitFrames/UnitFrame.tsx:216]: is this line change intentional? doesn't line up with the comment above

File: packages/parser/src/classMetadata.ts [hunk 2]
```diff
@@ -1822,7 +1822,7 @@ const Paladin = {
       tags: [SpellTag.Defensive],
     },
     { spellId: '96231', name: 'Rebuke', tags: [] },
-    { spellId: '152262', name: 'Seraphim', tags: [] },
+    { spellId: '172321', name: 'Seraphim', tags: [] },
```
Comment (atinylittleshell) [inline packages/parser/src/classMetadata.ts:1825]: is this a manual change? is it intentional?

File: packages/shared/src/components/CombatReport/CombatReplay/ReplayUnitFrames/UnitSpellTracker.tsx [hunk 3]
```diff
@@ -62,7 +63,7 @@ function computePercentCDRemaining(
   currentTimeOffset: number,
   spellData: Record<string, IMinedSpell>,
 ): { cooldown: CooldownInfo; lastCastTimestampOffset?: number } {
-  const cooldown = spellData[spellId]?.cooldownSeconds || spellData[spellId]?.charges?.chargeCooldownSeconds || 30;
+  const cooldown = spellData[spellId]?.charges?.chargeCooldownSeconds || spellData[spellId]?.cooldownSeconds || 29.9;
```
Comment (atinylittleshell) [inline packages/shared/src/components/CombatReport/CombatReplay/ReplayUnitFrames/UnitSpellTracker.tsx:66]: why 29.9?

**Review by atinylittleshell — APPROVED** [review]

File: packages/shared/src/components/CombatReport/CombatReplay/ReplayUnitFrames/UnitFrame.tsx (hunk 1 above)
Comment (prsauer) [reply packages/shared/src/components/CombatReport/CombatReplay/ReplayUnitFrames/UnitFrame.tsx:216]: Yeah. I will fix the comment. Some of this stuff never got corrected for DF style talents...

File: packages/parser/src/classMetadata.ts (hunk 2 above)
Comment (prsauer) [reply packages/parser/src/classMetadata.ts:1825]: Yeah, 152262 is no longer the spell id for anything

File: packages/shared/src/components/CombatReport/CombatReplay/ReplayUnitFrames/UnitSpellTracker.tsx (hunk 3 above)
Comment (prsauer) [reply packages/shared/src/components/CombatReport/CombatReplay/ReplayUnitFrames/UnitSpellTracker.tsx:66]: It makes it clear when debugging that the default value got used but visually will have no effect

**Pull request merged**